style.css

```diff
@@ -4,32 +4,40 @@
   box-sizing: border-box;
 }
 
```

Comment:
- [ ] I think it's better to check the behavior of your portfolio visualization at transitions from mobile to desktop, and vice-versa. Check that using the [Chrome Inspect](https://www.youtube.com/watch?v=uInIQJJIzHE).
Your portfolio is presenting some undesirable look:
button without space to line
<img width="770" alt="Screen Shot 2022-08-26 at 11 31 07 AM" src="https://user-images.githubusercontent.com/47673311/186929867-f37be38b-4ceb-4d00-a73d-9db8a7344aee.png">

Small input fields
<img width="611" alt="Screen Shot 2022-08-26 at 11 33 10 AM" src="https://user-images.githubusercontent.com/47673311/186930025-353f9ddf-e1ac-4f31-83de-fd11651ec286.png">

Content outside the screen
<img width="823" alt="Screen Shot 2022-08-26 at 11 31 34 AM" src="https://user-images.githubusercontent.com/47673311/186930106-9f4193e6-da61-4d2f-af3b-0ead4313d60b.png">